packages/wrangler/src/config.ts

```diff
@@ -204,6 +204,15 @@ export type Config = {
     preview_id?: string;
   }[];
 
+  r2_buckets?: {
+    /** The binding name used to refer to the R2 bucket in the worker. */
+    binding: string;
```

Comment:
we're moving to calling this "name" elsewhere, can we call it "name" here? did the implmentation in wrangler 1 call it "binding"? I think we can still rename this since we've not rolled it out.